### DISCUSSION TITLE
How to build gatsby repo when packages have been updated

### DISCUSSION THREAD

**Discussion by herecydev** [gatsbyjs/gatsby#26602] [Help]
## Summary

Trying to upgrade `ink` in the gatsby repo. I have run `gatsby-dev --packages gatsby-cli` and that is pulling across my local gatsby repo fine and any changes are synced correctly. However any packages that are upgraded are not being reflected correctly.

`gatsby-dev` seems to understand these have changed: "'ink' changed version from ^2.7.1 to ^3.0.4". But the `ink` package in node_modules isn't updated and is still left at an older version.

What am I doing wrong here?

## Relevant information

![image](https://user-images.githubusercontent.com/11328618/90975729-c08ddb80-e52e-11ea-8b01-047e5ae4e216.png)

**Comment by LekoArts** [comment]
cc @pieh
